### PR TITLE
Added Stylish Portfolio and Creative Theme

### PR DESCRIPTION
**Pull request by volny**

### CODE REVIEW

File: _posts/2015-09-08-stylish-portfolio.markdown [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title:  "Stylish Portfolio"
+date:   2015-09-08 01:03:00
```
Comment (gynter) [inline _posts/2015-09-08-stylish-portfolio.markdown]: Hello! Please remove excessive whitespaces and quotes from front matter:

```
title: Stylish Portfolio
date: 2015-09-08 01:03:00
```

File: _posts/2015-09-08-stylish-portfolio.markdown [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+homepage: https://github.com/volny/stylish-portfolio-jekyll
+download: https://github.com/volny/stylish-portfolio-jekyll/archive/master.zip
+demo: https://volny.github.io/stylish-portfolio-jekyll/
+author: Start Bootstrap
+thumbnail: stylish.png
```
Comment (gynter) [inline _posts/2015-09-08-stylish-portfolio.markdown]: The thumbnail filename should be consistent with the post filename, but without date prefix: `stylish-portfolio.png`.

File: _posts/2015-09-08-stylish-portfolio.markdown [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+license: Apache License Version 2.0
+license_link: https://github.com/volny/stylish-portfolio-jekyll/blob/master/LICENSE
+---
+
+Stylish Portfolio is a one page Bootstrap portfolio theme with off canvas navigation and smooth scrolling through content sections.
+
```
Comment (gynter) [inline _posts/2015-09-08-stylish-portfolio.markdown]: Please remove empty newline at the end of the post.

**Comment by volny** [comment]
@gynter done!

**Comment by gynter** [comment]
Thanks! @mattvh :shipit:

**Pull request merged**